rs/bitcoin/ckbtc/minter/ckbtc_minter.did

```diff
@@ -122,12 +122,22 @@ type PendingUtxo = record {
     confirmations: nat32;
 };
 
+// Number of nanoseconds since the Unix Epoch
+type Timestamp = nat64;
+
+type SuspendedUtxo = record {
+    utxo : Utxo;
+    reason : SuspendedReason;
+    earliest_retry: Timestamp;
+};
+
 type UpdateBalanceError = variant {
     // There are no new UTXOs to process.
     NoNewUtxos : record {
         current_confirmations: opt nat32;
         required_confirmations: nat32;
         pending_utxos: opt vec PendingUtxo;
+        suspended_utxos: opt vec SuspendedUtxo;
```

Comment:
Is this reasonable/desirable to have this field here?
I'm asking because it clashes a bit with the other fields of the record: A (clean) UTXO is pending if its current confirmation count is below the required number of confirmations. These things don't matter for suspended UTXOs.

Reply:
I think those UTXOs that can be retried should be visible to the user, otherwise how would they know when they can retry or that they can even retry? Do you see a better place where to put them?

Reply:
Is it only the suspended UTXO that can be retried? In that case, I agree that it makes sense!

Reply:
Discussed offline, all cleared.